src/components/card/new-vertical-resource-card.tsx

```diff
@@ -154,6 +152,7 @@ export const ResourceLink: React.FC<{
 }) => (
   <Link href={path}>
     <a
+      className={className}
```

Comment:
`NewVerticalResourceCard` accepts a className but it wasn't being applied anywhere